Gemfile

```diff
@@ -76,6 +76,7 @@ gem 'strong_parameters'
 gem 'delayed_job_active_record', '0.3.3'
 gem 'daemons'
 
+gem 'rack', '~>1.4.6'
```

Comment:
@ulferts according to the Security posting, `1.4.5` is the first version that contains the fix. I didn't know there was a `1.4.6` release out (it's not neither mentioned in the changelog nor tagged in the [rack repo](https://github.com/rack/rack)).